### DISCUSSION TITLE
Ability to set repository visibility when forking

### DISCUSSION THREAD

**Discussion by arunsathiya** [community/community#69886] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

It'd be nice to be able to set the visibility when creating a forked repository. 

[Originally created on the gh CLI tracker](https://github.com/cli/cli/issues/8174).

**Comment by ghostinhershell** [comment]
Hey @arunsathiya, 

Thanks so much for sharing this feedback with the community!
